src/util/__tests__/importHtml.ts

```diff
@@ -394,3 +399,145 @@ it('set cursor on last thought after importing multiple thoughts in non-empty cu
       - x
       - y`)
 })
+
+describe('DOM', () => {
+  beforeEach(async () => {
+    await createTestApp()
+  })
+  afterEach(cleanupTestApp)
+
+  it('should paste text properly that is copied from MacOS Notes.app', async () => {
+    /* eslint-disable*/
```

Comment:
Explanation for disabling the linter?

Reply:
There are irregular whitespaces in span tags.

`        <p class="p1"><span class="Apple-converted-space">  </span>- B</p>
`

Reply:
Got it. I recommend just disabling the specific rule rather than all rules.